### PR TITLE
87 split tutorial notebook into smaller parts

### PR DESCRIPTION
**Pull request by gschwend**

### CODE REVIEW

File: docs/notebooks/0_introduction.ipynb [hunk 1]
```diff
@@ -29,17 +36,11 @@
     "    - [How to upload a data product to the PZ Server](#how-to-upload-a-data-product-to-the-pz-server)\n",
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:36]: I don't know if it is a local problem, but the links for the "How to upload a data product to the PZ Server", 
"How to download a data product from the PZ Server", "How to get general info from PZ Server" and "How to display the metadata of a data product" sections are not taking me to the corresponding secions in the notebook.

File: docs/notebooks/0_introduction.ipynb [hunk 2]
```diff
@@ -61,46 +62,49 @@
...
+    "The PZ Server is developed and delivered as part of the in-kind contribution program BRA-LIN, from LIneA to the Rubin Observatory's LSST project. The service is hosted in the Brazilian IDAC, not directly connected to the [Rubin Science Platform (RSP)](https://data.lsst.cloud/). However, user authorization requires the same credentials as RSP. For comprehensive documentation about the PZ Server, please visit the [PZ Server's documentation page](https://linea-it.github.io/pz-lsst-inkind-doc/). There, you will also find an overview of all LIneA's contributions related to the PZ production. The internal documentation of the API functions is available on the [API's documentation page](https://linea-it.github.io/pzserver/html/index.html).  "
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:67]: It seems that the API's documentation page link is broken.

File: docs/notebooks/0_introduction.ipynb [hunk 3]
```diff
@@ -61,46 +62,49 @@
...
+    "- **Training Set:** Sample for training photo-z algorithms (tabular data). It usually contains magnitudes, errors, and true redshifts.\n",
+    "- **Photo-z Validation Results:** The Results of a photo-z validation procedure (free format). They usually contain photo-z estimates (single estimates and/or PDFs) of a validation set, photo-z validation metrics, validation plots, etc.\n",
+    "- Photo-z Table: This category is for the results of a photo-z estimation procedure. Ideally, the data should be in the same format as the photo-z tables delivered by the DM as part of the LSST data releases. If the data is larger than the file upload limit (200MB), the product entry will store only the metadata, and instructions on accessing the data should be provided in the description field. Storage space can be provided exceptionally for larger tables, depending on the science project justification (to be evaluated by IDAC's management committee).  "
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:83]: "Photo-z Table" is not in bold as others.

File: docs/notebooks/0_introduction.ipynb [hunk 4]
```diff
@@ -228,7 +229,6 @@
...
     "The object `pz_server` just created above can provide access to data and metadata stored in the PZ Server. It also brings useful methods for users to navigate through the available contents. The methods with the preffix `get_` return the result of a query on the PZ Server database as a Python dictionary, and are most useful to be used programatically (see detaials on the [API documentation page](https://linea-it.github.io/pzserver/html/index.html)). Alternatively, those with the preffix `display_` show the results as a styled [_Pandas DataFrames_](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html), optimized for Jupyter Notebook (note: column names might change in the display version). For instance:\n",
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:232]: I think there is a typo here. It should be "see details".

File: docs/notebooks/0_introduction.ipynb [hunk 5]
```diff
@@ -228,7 +229,6 @@
...
     "\n",
     "Display the list of product types supported with a short description;"
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:234]: "Display all data products available (WARNING: this list can rapdly grow during the survey's operation)." I think it is "rapidly"

File: docs/notebooks/0_introduction.ipynb [hunk 6]
```diff
@@ -531,7 +427,6 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The metadata of a given data product is the information provided by the user on the upload form. This information is attached to the data product contents and is available for consulting on the PZ Server page or using this Python API (`pzserver`). \n",
     "The metadata of a given data product is the information provided by the user on the upload form. This information is attached to the data product contents and is available for consulting on the PZ Server page or using this Python API (`pzserver`). \n",
     "\n",
     "All data products stored on PZ Server are identified by a unique **id** number or an unique name, a _string_ called **internal_name**, which is created automatically at the moment of the upload by concatenating the product **id** to the name given by its owner (replacing blank spaces by \"_\", lowering cases, and removing special characters). "
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:432]: Despite starting with a vowel, the "u" in unique has a consonant sound (something like "you"). So, the correct article is "a".

Below, in  "The time spent to download a data product depends on the internet connections between the user and the host", I think it should be just "connection".

File: docs/notebooks/0_introduction.ipynb [hunk 7]
```diff
@@ -610,7 +505,7 @@
     "\n",
     "https://pz-server-dev.linea.org.br/product/6_simple_training_set\n",
     "\n",
-    "<font color=red> WARNING:</font> The URL works only with the internal name, **not** with the **id** number. \n"
+    "<font color=red> WARNING:</font> The URL works only with the **complete internal name**, not with just the **id** number. \n"
    ]
   },
   {
```
Comment (luigilcsilva) [inline docs/notebooks/0_introduction.ipynb:511]: "The tabular data is alocated in the attribute `data`". I think it should be "allocated".

File: docs/notebooks/1_specz_catalogs.ipynb [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Introduction \n",
+    "\n",
+    "Welcome to the PZ Server tutorials. If you are reading this notebooks for the first time, we recommend to not skip the introduction notebook: `0_introduction.ipynb` also available in this same repository. \n"
```
Comment (luigilcsilva) [inline docs/notebooks/1_specz_catalogs.ipynb:31]: It should be "this notebook" or "these notebooks". In  "we recommend to not skip the introduction notebook", i think the correct sentence is "we recommend not to skip the introduction notebook".

File: docs/notebooks/1_specz_catalogs.ipynb [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+    "## Spec-z Catalogs "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "In the context of the PZ Server, Spec-z Catalogs are defined as any catalog containing spherical equatorial coordinates and spectroscopic redshift measurements (or, analogously, true redshifts from simulations). A Spec-z Catalog can include data from a single spectroscopic survey or a combination of data from several sources. To be considered as a single Spec-z Catalog, the data should be provided as a single file to PZ Server's the upload tool. For multi-survey catalogs, it is recommended to add the survey name or identification as an extra column. \n",
```
Comment (luigilcsilva) [inline docs/notebooks/1_specz_catalogs.ipynb:120]: "the data should be provided as a single file to PZ Server's the upload tool" -> "the data should be provided as a single file to the PZ Server's upload tool"

File: docs/notebooks/2_training_sets.ipynb [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Introduction \n",
+    "\n",
+    "Welcome to the PZ Server tutorials. If you are reading this notebooks for the first time, we recommend to not skip the introduction notebook: `0_introduction.ipynb` also available in this same repository. \n"
```
Comment (luigilcsilva) [inline docs/notebooks/2_training_sets.ipynb:31]: "this notebooks" -> "this notebook" or "these notebooks" | "we recommend to not skip" -> "we recommend not to skip"

File: docs/notebooks/2_training_sets.ipynb [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+    "* Subset Flag - `integer`, `float`, or `string`\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### PZ Server Pipelines\n",
+    "Trainind Sets can be uploaded by users on PZ Server website or via the `pzserver` library. Also, they can be created as the spatial cross-matching between a given Spec-z Catalog previously registered in the system and an Object table from a given LSST Data Release available in the Brazilian IDAC by the PZ Sever's pipeline \"Training Set Maker\" (under development). Any Training Set built by the pipeline is automaticaly registered as a regular user-generated data product and has no difference from the uploaded ones. \n",
```
Comment (luigilcsilva) [inline docs/notebooks/2_training_sets.ipynb:139]: "Trainind" -> "Training" | "automaticaly" -> "automatically"

File: docs/notebooks/1_specz_catalogs.ipynb [hunk 12]
```diff
@@ -0,0 +1,258 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### PZ Server Pipelines\n",
+    "Spec-z Catalogs can be uploaded by users on PZ Server website or via the `pzserver` library. Also, they can be created as the combination of a list of other Spec-z Catalogs previously registered in the system by the PZ Sever's pipeline \"Combine Spec-z Catalogs\" (under development). Any catalog built by the pipeline is automaticaly registered as a regular user-generated data product and has no difference from the uploaded ones. "
```
Comment (luigilcsilva) [inline docs/notebooks/1_specz_catalogs.ipynb:141]: "automaticaly" -> "automatically"

File: docs/notebooks/rst/output_22_1.png
Comment (luigilcsilva) [inline docs/notebooks/rst/output_22_1.png:1]: Should there be units?

Comment (gschwend) [reply docs/notebooks/rst/output_22_1.png:1]: will be fixed with issue #97

**Review by crisingulani — APPROVED** [review]

**Pull request merged**